server/plugin/plugin.go

```diff
@@ -67,10 +67,15 @@ func (p *Plugin) OnActivate() error {
 	}
 
 	p.httpHandler = http.NewHandler()
-	p.notificationHandler = api.NewNotificationHandler(p.newAPIConfig())
+
+	conf := p.newAPIConfig()
+	p.notificationHandler = api.NewNotificationHandler(conf)
 
 	command.Register(p.API.RegisterCommand)
 
+	// j := api.NewAvailabilityJob(api.New(conf, ""))
```

Comment:
Job is currently disabled. It should be easier for us to configure on and off.

Reply:
2/5 The job needs to be notified (or restarted) on configuration changes